packages-ts/starknet-gauntlet-ocr2/src/commands/ocr2/inspection/inspect.ts

```diff
@@ -3,11 +3,36 @@ import {
   IStarknetProvider,
   makeInspectionCommand,
 } from '@chainlink/starknet-gauntlet'
-import { InvokeTransactionReceiptResponse } from 'starknet'
+import { shortString, validateAndParseAddress } from 'starknet'
 import { CATEGORIES } from '../../../lib/categories'
 import { ocr2ContractLoader } from '../../../lib/contracts'
+import BN from 'bn.js'
 
-type QueryResult = {}
+type QueryResult = {
+  typeAndVersion: string
+  description: string
+  owner: string
+  decimals: number
+  latestConfigDetails: {
+    configCount: number
+    blockNumber: number
+    configDigest: string
+  }
+  transmitterInfo: {
+    transmitter: string
+    owedPayment: string
+  }[]
+  billing: {
+    observationPaymentGjuels: string
+    transmissionPaymentGjuels: string
+    gasBase: string
+    gasPerSignature: string
+  }
+  linkAvailableForPayment: {
+    isNegative: boolean
+    absoluteDifference: string
+  }
+}
```

Comment:
Assume this is all the required fields and expected formatting/types from the context given from the ticket - please let me know if anything was missed

Reply:
example output:
```
{
  typeAndVersion: 'ocr2/aggregator.cairo 1.0.0',
  description: 'test / test',
  owner: '0x0727664d8de5dc8b8d200bfa10685ab1826e4144713e6522db7dc5c5a9138578',
  decimals: 8,
  latestConfigDetails: {
    configCount: 1,
    blockNumber: 6,
    configDigest: '0x432de9deb179d411ebe03070a1b05a3ab95f99eb4ff50fcc7be4414f45a4f'
  },
  transmitterInfo: [
    {
      transmitter: '0x06da90a36b310e54c14ec686ea3d10fc44a14303d45b1f90d93589db8f99f3ef',
      owedPayment: '0'
    },
    {
      transmitter: '0x04737098fd5346ad65b322da48dbd310a1d00cd0d6609160599c8531d0839e92',
      owedPayment: '0'
    },
    {
      transmitter: '0x03490e8705187e7bd691f516d96737265c71bcac14d0deb53c5f81fb6266c68c',
      owedPayment: '0'
    },
    {
      transmitter: '0x018358bee8a222fa809fa392acc497d1423ed086fca91f93bcf71c8e7b4af2b4',
      owedPayment: '0'
    }
  ],
  billing: {
    observationPaymentGjuels: '0',
    transmissionPaymentGjuels: '0',
    gasBase: '0',
    gasPerSignature: '0'
  },
  linkAvailableForPayment: { isNegative: true, absoluteDifference: '0' }
}
```